server/app/lib/actions/fusor/deployment/rhev/ose_launch.rb

```diff
@@ -57,10 +57,10 @@ def run
                                                :proxy => Domain.find(1).proxy
                                              })
             if subdomain.valid?
+              ::Fusor.log.debug "====== OSE wildcard subdomain is not valid, it might conflict with a previous entry. Skipping. ======"
```

Comment:
Yeah it's just strange naming. It means the subdomain is a valid entry that already exists. So in this case if it already exists don't try to add a new one.

If the entry doesn't exist, we will create jt. This is temporary as we plan on having validation with the user to not give a duplicate name in the UI.